includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```
@@ -4351,6 +4361,28 @@ private function validate_field( string $field_key, $field_value ) {
+	 */
+	private function deprecated_property( string $property_name, string $version, string $replacement ): void {
+		if ( WP_DEBUG ) {
+			$message = sprintf(
+				'Property %1$s is <strong>deprecated</strong> since version %2$s! Use %3$s instead.',
+				$property_name,
+				$version,
+				$replacement
+			);
+			wp_trigger_error( '', $message, E_USER_DEPRECATED );
+		}
+	}
```

PHPMD (2.15.0)

[error] The method deprecated_property is not named in camelCase. (undefined)

(CamelCaseMethodName)